### ISSUE TITLE
Suggested feature: restore original window layout / position on unsnapping window

### ISSUE THREAD

**Issue by geoHeil** [eczarny/spectacle#510]
It would be great if the original window size and position could be restored on unspanning the window.

**Comment by 54mu3l** [comment]
Same here. This feature is required.
At the moment I do that with "Undo", but it doesn't work when the window is moved multiple times...

**Comment by denys** [comment]
Agree. Previously I have used another proprietary app for windows management and this was a very useful feature. If spectacle will support such that's will be awesome.

**Comment by eczarny** [comment]
This is interesting. Are we talking about a "reset" action? Something that just resets a window to where it was before Spectacle manipulated it?

**Comment by HighDensityAwesome** [comment]
I agree. Much like in GNOME after snapping an application window to fullscreen using CMD + Up and then you use the same shortcut again to snap it back to it's unchanged state, or perhaps back to a smaller state (centered maybe?).

**Comment by aop** [comment]
A reset would be awesome

**Comment by lifeinchords** [comment]
It would great to have this ability. I use Maximize shortcut a lot, after moving something between displays.. and often I'll want to un-maximize it later

**Comment by torgeir** [comment]
I would love this. Just moved here from Flexiglass, mainly for the keyboard-resizable-window-feature. Over there, hitting the same movement key twice (no matter which one) restores the previous window size and position as well - that would be awesome.

**Comment by bezhermoso** [comment]
It would love to see this as well. As reference, Magnet has a "Restore" feature that does exactly this. It would be great to see it in Spectacle as well.
